src/MedTrackerAPI.Tests/MedTrackerAPI.Tests.csproj

```diff
@@ -10,6 +10,7 @@
     </PropertyGroup>
 
     <ItemGroup>
+        <PackageReference Include="Bogus" Version="35.6.1" />
```

Comment:
My favorite package. I actually use Sonnecker.Utils.AutoBogus which will do automatic setups for you and then you can configure individual properties when needed.